configs/textdet/dbnet/dbnet_r18_fpnc_100k_iters_synthtext.py

```diff
@@ -0,0 +1,33 @@
+_base_ = [
+    '../../_base_/default_runtime.py',
+    '../../_base_/schedules/schedule_sgd_100k_iters.py',
+    '../../_base_/det_models/dbnet_r18_fpnc.py',
```

Comment:
Shall we create a config for r18dcn?

Reply:
> Shall we create a config for r18dcn?

Yes, I think so, since the official code use dcn. Shall we directly override the r18 with r18dcn or create another one?